lang/erlang/patches/010-openssl-md5.patch

```diff
@@ -0,0 +1,421 @@
+--- a/erts/emulator/openssl/crypto/md5/md5_local.h
++++ b/erts/emulator/openssl/crypto/md5/md5_local.h
```

Comment:
Why do we add patch without Git header? There are no details, why do we need to have it. Who is the author, etc.


Could you please take this patch as it should be: https://patch-diff.githubusercontent.com/raw/erlang/otp/pull/7989.patch ?

Reply:
Totally makes sense, done.